pkg/scheduler/metrics/metrics.go

```diff
@@ -257,12 +291,20 @@ var (
 
 var registerMetrics sync.Once
 
+// RegisterVolumeSchedulingMetrics is used for scheduler, because the volume binding cache is a library
+// used by scheduler process.
+func RegisterVolumeSchedulingMetrics() {
+	legacyregistry.MustRegister(VolumeBindingRequestSchedulerBinderCache)
+	legacyregistry.MustRegister(VolumeSchedulingStageLatency)
+	legacyregistry.MustRegister(VolumeSchedulingStageFailed)
+}
+
 // Register all metrics.
 func Register() {
 	// Register the metrics.
 	registerMetrics.Do(func() {
 		RegisterMetrics(metricsList...)
-		volumeschedulingmetrics.RegisterVolumeSchedulingMetrics()
```

Comment:
This function is only called by the scheduler, so it can be removed.

The metrics definitions are also only used in `pkg/controller/volume/scheduling/scheduler_binder.go` and `pkg/controller/volume/scheduling/metrics/metrics.go`, so they can be updated to refer to the new definitions in `pkg/scheduler/metrics/metrics.go` (and so their definitions also removed).

With this, I believe the entire `pkg/controller/volume/scheduling/metrics` package can be removed